ui/js/page/settings/view.jsx

```diff
@@ -252,6 +278,49 @@ class SettingsPage extends React.PureComponent {
             </div>
           </div>
         </section>
+
+        <section className="card">
+          <div className="card__content">
+            <h3>{__("Key Fee")}</h3>
+          </div>
+          <div className="card__content">
+            <FormField
+              ref="max_key_fee"
+              type="number"
+              min="0"
+              placeholder="0"
```

Comment:
Ok. But our suggested value is US$ 50 as of [this](https://github.com/lbryio/lbry/pull/736) PR, so should I change this to 50 instead?